.travis.yml

```diff
@@ -1,35 +1,39 @@
 language: php
 
+php:
+  - 7.1
+  - 7.2
+  - 7.3
+
 env:
   global:
     - SETUP=stable
+  matrix:
+    - laravel=5.6.*
+    - laravel=5.7.*
+    - laravel=5.8.*
+    - laravel=dev-master
+    - SETUP=nightly laravel=5.6.*
+    - SETUP=nightly laravel=5.7.*
+    - SETUP=nightly laravel=5.8.*
+    - SETUP=nightly laravel=dev-master
 
 matrix:
   fast_finish: true
-  include:
-    - php: 7.1
-    - php: 7.1
-      env: SETUP=nightly
-    - php: 7.2
-    - php: 7.2
-      env: SETUP=nightly
-    - php: 7.3
-    - php: 7.3
-      env: SETUP=nightly
   allow_failures:
-    - php: 7.1
-      env: SETUP=nightly
-    - php: 7.2
-      env: SETUP=nightly
-    - php: 7.3
-      env: SETUP=nightly
+    - env: laravel=dev-master
+    - env: SETUP=nightly laravel=5.6.*
+    - env: SETUP=nightly laravel=5.7.*
+    - env: SETUP=nightly laravel=5.8.*
+    - env: SETUP=nightly laravel=dev-master
 
 sudo: false
 
 before_install:
   - phpenv config-rm xdebug.ini || true
 
 install:
+  - composer require "laravel/framework=${laravel}" --prefer-dist --no-interaction --no-suggest
```

Comment:
Hmm. I'm not sure this is the wtg (see failing tests)

Reply:
Fixed.